### ISSUE TITLE
some errors about train.py

### ISSUE THREAD

**Issue by vincennnnt** [MathsXDC/DaSiamRPNWithOfflineTraining#3]
Sorry to interrupt u,and i really thank u for ur work. when i run train.py, i got some questions ,'xxxx' object has no attribute 'to' .Is that caused by the version of pytorch.The version of torch is 0.3.1?

**Comment by vincennnnt** [comment]
And line [103]～[149]inds_inside and total_anchors are not defined

**Comment by MathsXDC** [comment]
@vincennnnt The prerequisites have been updated and these bugs fixed. More discussions are welcome.

**Comment by davidsmithwj** [comment]
![image](https://user-images.githubusercontent.com/25165279/55119917-b2b40b00-512e-11e9-8de1-4ec81ccb8376.png)
when running the code, the cls_loss will be nan. I think it is because of the 'mask' in train.py can be zero. But I am not sure. And another question is that when running the code, there will be a error "cuda runtime error: an illegal memory access was encountered". hoping to hear from you soon. thank you very much
